src/current/v24.3/restore.md

```diff
@@ -150,6 +150,8 @@ CockroachDB uses the URL provided to construct a secure API call to the service
 
 ## Functional details
 
+CockroachDB uses checksums to ensure data integrity during both the write and read processes. When data is written to disk, CockroachDB calculates and stores checksums for each block of data. The checksums are then used to verify the integrity of the data when it is read back from disk. A restore job reads each block of data from the external backup storage and verifies the checksum to ensure that the data has not been corrupted during the storage or transfer of the backup data.
```

Comment:
A "block" has a very specific meaning:  a part of an SST. I'm unsure if you need to explain this.

Reply:
I imagine most users will interpret "block" simply as a grouping of data. But is there another phrasing you think might be better?

Reply:
@msbutler @benbardin Do you think it would be better to go more general in the restore docs:

> CockroachDB uses checksums to ensure data integrity during both the write and read processes. When data is written to disk, CockroachDB calculates and stores checksums with the data. The checksums are then used to verify the integrity of the data when it is read back from disk. A restore job reads the data from the external backup storage and verifies the checksums to ensure that the data has not been corrupted during the storage or transfer of the backup data.

And we can link out to our [reference material SSTs](https://www.cockroachlabs.com/docs/stable/architecture/storage-layer#ssts) in case readers want to get more detailed? I think that Ben is right that most people are likely to interpret "block of data" as a general group of data. 

I'm also OK to get more specific. Another option is to have this general paragraph here on the restore page, and then add a similar paragraph to the [Backup Architecture](https://www.cockroachlabs.com/docs/stable/backup-architecture) docs with the added detail re "block is part of an SST".

Reply:
i like the more general approach!

Reply:
I'm comfortable with either so let's do michael's!